server/app.js

```diff
@@ -0,0 +1,24 @@
+var express = require('express'),
+    app = express(),
+    waypointRouter = express.Router(),
+    authenticationRouter = express.Router();
+
+
+require('./routers/waypointRouter')(waypointRouter);
+require('./routers/authenticationRouter')(authenticationRouter);
+
+
+// requests to /waypoint/* will be sent to this router
+
+app.use('/waypoint', waypointRouter);
+app.use('/user', authenticationRouter);
+
+// catch 404 and forward to error handler
```

Comment:
Please start comments with caps, and format like you're reading with "It" in front
e.g. 
// Catches 404 and forwards to error handler